### PR TITLE
Deploy using Google Cloud Run and add Google Analytics

### PR DESCRIPTION
**Pull request by igorsyl**
This is _almost_ working at [sbtc.world](https://sbtc.world)

### CODE REVIEW

**Comment by radicleart** [comment]
Deployment seems to be working but the web app is trying to connect to https://mainnet.api.sbtc.world/bridge-api/ and receiving;

net::ERR_NAME_NOT_RESOLVED

**Comment by igorsyl** [comment]
> 

Thanks! Let me update to new API endpoint.

File: svelte.config.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // import adapter from '@sveltejs/adapter-node';
 import adapter from '@sveltejs/adapter-static';
 import preprocess from 'svelte-preprocess';
-const dev = process.argv.includes('dev') || process.argv.includes('build-stag');
+// const dev = process.argv.includes('dev') || process.argv.includes('build-stag');
```
Comment (sergey-shandar) [inline svelte.config.js:4]: Why did we comment this?

File: .idea/.gitignore
Comment (sergey-shandar) [inline .idea/.gitignore:1]: I think, we should ignore all files in the `.idea/`folder.

File: svelte.config.js (hunk 1 above)
Comment (igorsyl) [reply svelte.config.js:4]: not used. need to remove.

File: .idea/.gitignore
Comment (igorsyl) [reply .idea/.gitignore:1]: Please keep if possible!

**Review by sergey-shandar — APPROVED** [review]

**Review by sergey-shandar — APPROVED** [review]

**Comment by radicleart** [comment]
@sergey-shandar has this been superceded by https://github.com/Trust-Machines/sbtc-bridge-web/pull/93 ?

I thiink we want to close without merging ?

**Comment by igorsyl** [comment]
> @sergey-shandar has this been superceded by #93 ?
> 
> I thiink we want to close without merging ?

@radicleart let's close this together when we meet again. pls book time w/me.

**Comment by radicleart** [comment]
Closing this PR - it has been absorbed by [PR 94](https://github.com/Trust-Machines/sbtc-bridge-web/pull/94)

**Pull request closed without merging**